src/api-service/__app__/onefuzzlib/tasks/defs.py

```diff
@@ -21,6 +21,7 @@
             TaskFeature.target_options,
             TaskFeature.target_timeout,
             TaskFeature.coverage_filter,
+            TaskFeature.target_must_use_input,
```

Comment:
Should we add this to the `analysis` task, too?

Reply:
I didn't include the analysis task, because it could be sent through analysis_env or analysis_options.